09_few-to-no-labels.ipynb

```diff
@@ -2702,9 +2702,12 @@
     "inputs = tokenizer(\"Transformers are awesome!\", return_tensors=\"np\")\n",
     "outputs = data_collator([{\"input_ids\": inputs[\"input_ids\"][0]}])\n",
     "\n",
+    "original_input_ids = inputs[\"input_ids\"][0]\n",
+    "masked_input_ids = outputs[\"input_ids\"][0]\n",
+    "\n",
     "pd.DataFrame({\n",
-    "    \"Original tokens\": tokenizer.convert_ids_to_tokens(inputs[\"input_ids\"][0]),\n",
-    "    \"Masked tokens\": tokenizer.convert_ids_to_tokens(outputs[\"input_ids\"][0]),\n",
+    "    \"Original tokens\": tokenizer.convert_ids_to_tokens(original_input_ids),\n",
+    "    \"Masked tokens\": tokenizer.convert_ids_to_tokens(masked_input_ids),\n",
     "    \"Original input_ids\": original_input_ids,\n",
     "    \"Masked input_ids\": masked_input_ids,\n",
```

Comment:
The ```original_input_ids``` and ```masked_input_ids``` were referenced in lines 2708 and 2709 to construct the DataFrame but never defined.